stdlib/ballerina-http/src/main/ballerina/http/Package.md

```diff
@@ -36,6 +36,43 @@ See [Mutual SSL Example](https://ballerinalang.org/docs/by-example/mutual-ssl).
 
 See [Caching Example](https://ballerinalang.org/docs/by-example/caching), [HTTP Disable Chunking Example](https://ballerinalang.org/docs/by-example/http-disable-chunking).
 
+###WebSockets
+The package also provides support for WebSockets. There are two types of WebSocket endpoint which are `WebSocketClient` and `WebSocketListener`. Both endpoints support all WebSocket frames. The `WebSocketClient` has a callback service.
```

Comment:
Change "There are two types of WebSocket endpoint which are `WebSocketClient` and `WebSocketListener`." to "There are two types of WebSocket endpoints: `WebSocketClient` and `WebSocketListener`."